pkgs/applications/networking/seafile-client/default.nix

```
@@ -1,23 +1,44 @@
-{ mkDerivation, lib, fetchFromGitHub, pkg-config, cmake, qtbase, qttools
-, seafile-shared, jansson, libsearpc
-, withShibboleth ? true, qtwebengine }:
+{ mkDerivation
+, lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qtbase
+, qttools
+, libuuid
+, seafile-shared
+, jansson
+, libsearpc
+, withShibboleth ? true
+, qtwebengine
+, wrapQtAppsHook
+}:
 
-mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "seafile-client";
-  version = "8.0.7";
+  version = "9.0.2";
 
   src = fetchFromGitHub {
     owner = "haiwen";
     repo = "seafile-client";
-    # Tag v8.0.7 used to point to this commit, but was re-tagged later to a state
-    # that fails to link properly
-    rev = "8b96abf64e6aa19e3beeff6b322bcd6d28120bdd";
-    sha256 = "00wfr7dvbyl7pg1xgssgz8a94c7c4n5r9266lhy9qcbz456hdcgj";
+    rev = "v${version}";
+    sha256 = "sha256-6AsvqlXDmTr3yBvYlV/0qjH+KnZ03S5TU7yzGt0MqiI=";
   };
 
-  nativeBuildInputs = [ pkg-config cmake ];
-  buildInputs = [ qtbase qttools seafile-shared jansson libsearpc ]
-    ++ lib.optional withShibboleth qtwebengine;
+  nativeBuildInputs = [
+    libuuid
+    pkg-config
+    cmake
+    wrapQtAppsHook
+    qttools
+  ];
+
+  buildInputs = [
+    seafile-shared
+    jansson
+    libsearpc
+  ] ++ lib.optional withShibboleth qtwebengine;
 
   cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ]
     ++ lib.optional withShibboleth "-DBUILD_SHIBBOLETH_SUPPORT=ON";
```

`-DCMAKE_BUILD_TYPE=Release` by default, see https://github.com/NixOS/nixpkgs/blob/5ebc55097f8134e6247dfb17808c8b732d4c7e8e/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L111
```suggestion
  cmakeFlags = lib.optional withShibboleth "-DBUILD_SHIBBOLETH_SUPPORT=ON";
```